forui/lib/src/widgets/accordion.dart

```diff
@@ -0,0 +1,399 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'dart:math' as math;
+
+import 'package:forui/forui.dart';
+import 'package:forui/src/foundation/tappable.dart';
+import 'package:forui/src/foundation/util.dart';
+import 'package:meta/meta.dart';
+
+/// A controller that stores the expanded state of an [FAccordion].
+class FAccordionController extends ChangeNotifier {
```

Comment:
For consistency, I think "expand" should be replaced with "show" while "close" should be replaced with "hide".